.github/workflows/build-and-test.yaml

```diff
@@ -0,0 +1,25 @@
+name: Build and Test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
```

Comment:
Should we add a `name` for the job as a convention?

Reply:
its called build, don't think we have a convention yet.